mindsdb/api/executor/datahub/datanodes/information_schema_datanode.py

```diff
@@ -344,31 +57,7 @@ def __init__(self, session):
                 integration_controller=self.session.integration_controller,
             )
 
-        self.get_dataframe_funcs = {
-            "TABLES": self._get_tables,
-            "COLUMNS": self._get_columns,
-            "SCHEMATA": self._get_schemata,
-            "ENGINES": self._get_engines,
-            "CHARACTER_SETS": self._get_charsets,
-            "COLLATIONS": self._get_collations,
-            "MODELS": self._get_models,
-            "MODELS_VERSIONS": self._get_models_versions,
-            "DATABASES": self._get_databases,
-            "ML_ENGINES": self._get_ml_engines,
-            "HANDLERS": self._get_handlers,
-            "JOBS": self._get_jobs,
-            "JOBS_HISTORY": self._get_jobs_history,
-            "MDB_TRIGGERS": self._get_triggers,
-            "CHATBOTS": self._get_chatbots,
-            "KNOWLEDGE_BASES": self._get_knowledge_bases,
-            "SKILLS": self._get_skills,
-            "AGENTS": self._get_agents
-        }
-        for table_name in self.information_schema:
-            if table_name not in self.get_dataframe_funcs:
-                self.get_dataframe_funcs[table_name] = partial(
-                    self._get_empty_table, table_name
-                )
+        self.tables = {t.name: t() for t in self.tables_list}
```

Comment:
Looks like tables classes do not use `self`. May be make them static? so we do not need to instantiate them each time.

Reply:
changed to class_methods